src/main/java/com/microsoft/azure/functions/worker/handler/WorkerInitRequestHandler.java

```diff
@@ -26,7 +29,15 @@ String execute(WorkerInitRequest request, WorkerInitResponse.Builder response) {
         response.putCapabilities("RpcHttpTriggerMetadataRemoved", "RpcHttpTriggerMetadataRemoved");
         response.putCapabilities("HandlesWorkerTerminateMessage", "HandlesWorkerTerminateMessage");
         response.putCapabilities("HandlesWorkerWarmupMessage", "HandlesWorkerWarmupMessage");
+
+        if (Boolean.parseBoolean(System.getenv(JAVA_ENABLE_OPENTELEMETRY)) ||
+                Boolean.parseBoolean(System.getenv(JAVA_APPLICATIONINSIGHTS_ENABLE_TELEMETRY))) {
+            response.putCapabilities("WorkerOpenTelemetryEnabled", "true");
+            response.putCapabilities("WorkerApplicationInsightsLoggingEnabled", "true");
```

Comment:
[Double Checking] If JAVA_ENABLE_OPENTELEMETRY is true then do you need to set Capabilities "WorkerApplicationInsightsLoggingEnabled" to "true"?